uint8 LEFT=0
uint8 RIGHT=1	

uint8 side 
duckietown_msgs/Vector2D[2] pixels_line

geometry_msgs/Point[2] points
